### DISCUSSION TITLE
Add wealth projections

### DISCUSSION THREAD

**Discussion by svenstaro** [ananthakumaran/paisa#186] [Ideas]
I think it would be great if this software had some kind of wealth extrapolation/projection based on current trends. Maybe even taking into account the stock market trends in additional to `income - expenses`.

**Comment by ananthakumaran** [comment]
Have you checked the Goals? It has some crude projection functionality.

**Reply by svenstaro** [reply]
I didn't see this before. I'll be honest, I don't really this as a goal specifically. I just want the tool to be able to project x months/years into the future without a specific goal in mind. I want to see a graph of wealth accumulation over time, ideally.

EDIT: I just wanna add that the tool is really cool already!

**Reply by ananthakumaran** [reply]
Play around with a retirement goal or savings goal without a target date, it does some projection if possible using arima model if you don't specify the rate of return.

**Reply by svenstaro** [reply]
Will do, thanks!
